backend/census_historical_migration/management/commands/csv_to_postgres.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import boto3
+import csv
+
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.apps import apps
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
+
+census_to_gsafac_models = list(
+    apps.get_app_config("census_historical_migration").get_models()
+)
+census_to_gsafac_model_names = [m._meta.model_name for m in census_to_gsafac_models]
+s3_client = boto3.client(
+    "s3",
+    aws_access_key_id=settings.AWS_PRIVATE_ACCESS_KEY_ID,
+    aws_secret_access_key=settings.AWS_PRIVATE_SECRET_ACCESS_KEY,
+    endpoint_url=settings.AWS_S3_ENDPOINT_URL,
+)
+census_to_gsafac_bucket_name = settings.AWS_CENSUS_TO_GSAFAC_BUCKET_NAME
+DELIMITER = ","
+
+
+class Command(BaseCommand):
+    help = """
+        Populate Postgres database from csv files
+        Usage:
+        manage.py csv_to_postgres --folder <folder_name> --clean <True|False>
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument("--folder", help="S3 folder name")
+        parser.add_argument("--clean")
+        parser.add_argument("--sample")
+        parser.add_argument("--load")
```

Comment:
Would be helpful to flesh these options out by including `help`/`type`/`required`/`default` params